File: src/main/java/com/gymtracker/gym/exerciseLogs/dto/ExerciseLogResponse.java
package com.gymtracker.gym.exerciseLogs.dto;

import java.math.BigDecimal;
import java.time.LocalDateTime;

public record ExerciseLogResponse(
        Long id,
        String exerciseName,
        Integer setNumber,
        Integer reps,
        BigDecimal weightKg,
        LocalDateTime loggedAt
) {}

File: src/main/kotlin/com/gymtracker/gym/exerciseLogs/dto/ExerciseLogResponse.kt
package com.gymtracker.gym.exerciseLogs.dto

import java.math.BigDecimal
import java.time.LocalDateTime

data class ExerciseLogResponse (
    val id: Long,
    val exerciseName: String,
    val setNumber: Int,
    val reps: Int?,
    val weightKg: BigDecimal?,
    val loggedAt: LocalDateTime
)